### PR TITLE
JB 15-16

### PR DESCRIPTION
**Pull request by maijapapaya**
@jaredh159 @Henderjay

### CODE REVIEW

File: diary-alexander-jaffray/modernized/32-ch-16-scotland.adoc [hunk 1]
```diff
@@ -267,6 +272,9 @@ It affords some clear indications of a sound and living faith,
 an unsophisticated simplicity of love, a heavenly meekness and gratitude,
 which were now fast ripening for a better state.
 
+[.embedded-content-document.epistle]
```
Comment (maijapapaya) [inline diary-alexander-jaffray/modernized/32-ch-16-scotland.adoc:275]: Ok so this first paragraph here is embdded in the ODT and looks like it should be in the PDF as well, but doesn't seem to really be part of the epistle. I wonder if it was just a quote from the "Friends of Aberdeen" that didn't get ended before they inserted the epistle? Or is it taken from some other writing as part of the epistle? I just can't tell from any of the context either. I tried to attach a picture, but it didn't seem to work. It's on page 359 in the PDF and page 243 of the ODT.

File: diary-alexander-jaffray/modernized/32-ch-16-scotland.adoc [hunk 2]
```diff
@@ -310,13 +321,21 @@ So, dear Friends, in a fresh stream of Life, does my soul dearly salute you all,
 desiring the continuance of your prayers for me: so,
 I remain your Friend and Brother in the Lord,
 
-Peter Gardiner.`"
+[.signed-section-signature]
+Peter Gardiner.
+
+[.postscript]
```
Comment (maijapapaya) [inline diary-alexander-jaffray/modernized/32-ch-16-scotland.adoc:328]: Also here I guess isn't part of the epistle either. Maybe I missed why these were included in his actual epistle.

File: diary-alexander-jaffray/modernized/32-ch-16-scotland.adoc [hunk 3]
```diff
@@ -329,8 +348,13 @@ the spiritual welfare of which, we have been engaged in contemplating;
 he is at length about to put in force, by patient resignation unto death,
 that injunction of the Apostle, "`We ought to lay down our lives for the brethren.`" John 3:16.
 
+[.embedded-content-document.letter]
```
Comment (maijapapaya) [inline diary-alexander-jaffray/modernized/32-ch-16-scotland.adoc:352]: So here begins a letter that has letters within. I did what I thought looked good, but idk if it's right! It seems that John Bowstead wrote a letter to Francis Stamper about Peter Gardiner, and inserted a couple of Peter's letters. At least that's what I could make of it. :)

File: diary-alexander-jaffray/modernized/31-ch-15-scotland.adoc [hunk 4]
```diff
@@ -331,54 +346,57 @@ And all the creation is upheld by his Word and power,
...
-as well as for a testimony to the power and goodness
-of God in raising him up to his church.`"
+as well as __for a testimony to the power and goodness
+of God in raising him up to his church.__`"
 It is prepared from documents,
 put forth by those contemporaries of Robert Barclay who knew him well,
-and appears in the pages of A Short Account of His Life and Writings.
+and appears in the pages of [.book-title]#A Short Account of His Life and Writings.#
 
 He was distinguished by strong mental powers, particularly by great penetration,
```
Comment (Henderjay) [inline diary-alexander-jaffray/modernized/31-ch-15-scotland.adoc:403]: So, from here to the end of the chapter is a quote from the book A Short Account of the Life and Writings of Robert Barclay, by John G. Bevan. So, I think these three paragraphs should be a block quote. 👍🏻

**Review by Henderjay — CHANGES_REQUESTED** [review]
I saw your notes on chapter 16. It was a tricky chapter because there was a letter that appeared within a note from a meeting. And then later on (like you thought) there were a few letters from Peter Gardiner stuck inside a letter from John Bowstead. We don't really have a convention for how to deal with letters within letters, but I think what you did looks as good as anything I could think of.  👍🏻. The only thing I'm going to have you change is actually in chapter 15, where I left the note. That chunk should probably be a block quote. (I only marked it in the original, but it will need to be a block quote in both modernized and original).

**Comment by Henderjay** [comment]
great, merged!

**Pull request merged**